### DISCUSSION TITLE
A couple of feature requests

### DISCUSSION THREAD

**Discussion by PatWalters** [cbouy/mols2grid#4] [Ideas]
This is an awesome tool, here are a couple of features that I would find useful.  
1. Checkboxes for selections and the ability to query the grid from Python to get the selected molecules.  
2. SMARTS searching in the grid.  This could take two forms, either highlighting substructure matches or filtering, like the existing search capability.  It might be nice to have an API call that would switch the behavior of the search box from text searching to SMARTS search.

**Comment by cbouy** [comment]
@PatWalters Both features are now added in [v0.0.3](https://github.com/cbouy/mols2grid/releases/tag/v0.0.3) !
You can check the [demo](https://nbviewer.jupyter.org/github/cbouy/mols2grid/blob/master/demo.ipynb) notebook for an example
